modules/metastore/modules/metastore_facets/composer.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
We need this to be added to the DKAN composer.json, this won't get read be composer for a project

Reply:
This is an optional module that is available but not enabled by default, I think the follow up ticket is to create a second ddev site install command for a standing up a non-decoupled FE site. Or am I missing your point here?